scripts/generate_multimodal_eval_fixtures.py

```
@@ -0,0 +1,279 @@
+$graphics.DrawString("Total Due NOK 120000", $bodyFont, $brush, 70, 220)
+$graphics.DrawString("Payment due 14 April 2026", $bodyFont, $brush, 70, 290)
+$graphics.DrawString("Approval owner Finance team", $bodyFont, $brush, 70, 360)
+$bitmap.Save($OutputPath, [System.Drawing.Imaging.ImageFormat]::Jpeg)
+$titleFont.Dispose()
+$bodyFont.Dispose()
+$graphics.Dispose()
+$bitmap.Dispose()
+            """.strip(),
+            encoding="utf-8",
+        )
+        result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)